Client/src/Components/Blurb.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Design consideration: You could make one component of the box and just reuse it to create the others. This could be useful & creates re-usable component depending on how the other pages look like.

Is it possible to get it the blurbs aligned?